vendor/engines/sanger_sequencing/spec/presenters/sanger_sequencing/well_plate_presenter_spec.rb

```diff
@@ -59,5 +63,26 @@
         expect(sample_rows[8]).to match(["A03", samples[7].id.to_s, samples[7].customer_sample_id.to_s] + extra_columns)
       end
     end
+
+    context "when alternative_csv_format is enabled", feature_setting: { alternative_csv_format: true } do
+      before do
+        allow(SettingsHelper).to receive(:feature_on?).and_return(true) 
```

Comment:
We shouldn't need to do this, just `feature_setting` should be enough. Was the Feature Flag not set?